test/src/unit-azure.cc

```diff
@@ -45,6 +45,20 @@
 using namespace tiledb::common;
 using namespace tiledb::sm;
 
+using ConfMap = std::map<std::string, std::string>;
+using ConfList = std::vector<ConfMap>;
+
+static ConfList test_settings = {
+    {{"vfs.azure.storage_account_name", "devstoreaccount1"},
+     {"vfs.azure.storage_account_key",
+      "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/"
+      "K1SZFPTOtr/KBHBeksoGMGw=="},
+     {"vfs.azure.blob_endpoint", "127.0.0.1:10000/devstoreaccount1"}},
+    //{{"vfs.azure.storage_account_name", "devstoreaccount2"},
```

Comment:
Shall we add a TODO/NOTE inline that its disabled because of emulator issues?

Reply:
Good point, done.